packages/realm/src/Realm.ts

```diff
@@ -147,7 +147,6 @@ type ObjectSchemaExtra = {
 
 // Using a set of weak refs to avoid prevention of garbage collection
 const RETURNED_REALMS = new Set<binding.WeakRef<binding.Realm>>();
-const NOT_VERSIONED = 18446744073709551615n;
```

Comment:
Nice to get rid of this 🙂

Reply:
Yeah. It is really just the unsigned 64-bit equivalent of -1 (same as the maximum uint64). I figured -1 makes a better sentinel value.